lib/Star/Kata/Infrastructure/KataInfrastructure.php

```diff
@@ -0,0 +1,33 @@
+<?php
+/**
+ * This file is part of the phpkata project.
+ *
+ * (c) Yannick Voyer (http://github.com/yvoyer)
+ */
+
+namespace Star\Kata\Infrastructure;
+
+use Star\Kata\KataDomain\KataService;
+use Star\Kata\Model\KataRepository;
+
+/**
+ * Class KataInfrastructure
+ *
+ * @author  Yannick Voyer (http://github.com/yvoyer)
+ *
+ * @package Star\Kata\Infrastructure
+ */
+interface KataInfrastructure
```

Comment:
this `interface` does not seem very useful. What is the significance of modelling the `Kata` `Infrastructure`?